util/addrmapgen.py

```diff
@@ -213,7 +213,7 @@ def main(file: str, outdir: pathlib.Path, tex_filename: str, md_filename: str):
     nr_q_entries = len(all_quadrant_entries)
 
     # Quadrant latex
-    NR_CLUSTER_ADDR_RANGES = 2  # tcdm and periphs
+    NR_CLUSTER_ADDR_RANGES = 3  # tcdm and periphs
     NR_CLUSTERS_PER_QUADRANT = 4  # hardcoded for now
```

Comment:
Unrelated to this PR, but I noticed that the hardcoded `NR_CLUSTERS_PER_QUADRANT ` breaks the generator if you try out smaller configurations (I think because of zero divides). Would this be difficult to change?

Reply:
Yeah, that was not tested. However, I think it should be a rather easy fix. Do you want to use it for the Dory trial setup? If so, do you have a branch where I could fix it?